src/api/getMetrics.ts

```diff
@@ -1,12 +1,12 @@
 import fetch from 'cross-fetch';
 
-import { GroupedMetrics, Metrics } from '../types/Metrics';
-
+import { APIKey } from '../types';
+import { GroupedMetrics, Metrics, MetricsGroup } from '../types/Metrics';
 import { formatUrl } from '../utils/url';
 import { handleResponse } from '../utils/handleResponse';
 
 
-export interface GetMetricsOptions {
+export type GetMetricsOptions<Group extends MetricsGroup> = {
```

Comment:
This is an object notation, so why should this be a `type` rather than an `interface`?

Reply:
I think using a consistent use of `type` (see other types) would be better